src/overlays/actors/ovl_Boss_Ganon/z_boss_ganon.c

```diff
@@ -108,127 +98,4928 @@ static ColliderCylinderInit D_808E4C2C = {
+            this->unk_278.x = this->bodyPartsPos[0].x;
+            this->unk_278.y = this->bodyPartsPos[0].y + 50.0f + 30.0f;
+            this->unk_278.z = this->bodyPartsPos[0].z;
+
+            Math_ApproachF(&this->unk_284, 0.25f, 0.1f, 0.006f);
+            Math_ApproachF(&this->unk_288, 255.0f, 1.0f, 255.0f);
+            Math_ApproachF(&this->unk_28C, 0.25f, 0.1f, 0.006f);
+
+            if ((this->timers[0] > 20) && (this->timers[0] < 60)) {
+                Math_ApproachF(&this->unk_290, 255.0f, 1.0f, 15.0f);
+            }
+
+            if (this->timers[0] == 0) {
+                this->fwork[FWORK_1] = Animation_GetLastFrame(&gDorfBigMagicWindupAnim);
+                Animation_MorphToPlayOnce(&this->skelAnime, &gDorfBigMagicWindupAnim, 0.0f);
+                this->unk_1C2 = 3;
+                this->timers[0] = 6;
+                this->timers[1] = 15;
+                Audio_PlayActorSound2(&this->actor, NA_SE_EN_GANON_DARKWAVE);
+                break;
+            }
+
+            Math_ApproachS(&this->actor.shape.rot.y, this->actor.yawTowardsPlayer, 5, 0x3E8);
+
+            if (this->timers[0] < -4) {
+                for (i = 0; i < ARRAY_COUNT(this->unk_294); i++) {
+                    Math_ApproachF(&this->unk_294[i], 0.0f, 1.0f, 40.0f);
+                }
+            } else if ((this->timers[0] >= 7) && (this->timers[0] < 26)) {
+                if (this->unk_1AC < ARRAY_COUNT(this->unk_294)) {
+                    this->unk_1AC++;
+                }
+
+                for (i = 0; i < this->unk_1AC; i++) {
+                    Math_ApproachF(&this->unk_294[i], 200.0f, 1.0f, 40.0f);
+                }
+            }
+
+            if (this->timers[0] <= 30) {
+                Math_ApproachF(&this->unk_284, 0.4f, 0.5f, 0.017f);
+                this->unk_28C = this->unk_284;
+            }
+
+            if (this->timers[0] <= 30) {
+                Math_ApproachF(&this->unk_2D0, 45.0f, 0.1f, 10.0f);
+                this->lensFlareTimer = 1;
+                this->lensFlareMode = 2;
+                gCustomLensFlarePos = this->unk_278;
+            }
+
+            if (this->timers[0] == 47) {
+                this->unk_274 = 1;
+            }
+
+            if (this->timers[0] == 46) {
+                this->unk_274 = 2;
+            }
+
+            if (this->timers[0] == 45) {
+                this->unk_274 = 3;
+            }
+
+            if (this->timers[0] == 44) {
+                this->unk_274 = 4;
+            }
+
+            if (this->timers[0] == 43) {
+                this->unk_274 = 5;
+            }
+
+            if (this->timers[0] == 42) {
+                this->unk_274 = 6;
+            }
+
+            if (this->timers[0] > 30) {
+                sp74.x = 0.0f;
+                sp74.y = Rand_ZeroFloat(10.0f) + 150.0f;
+                sp74.z = 0.0f;
+
+                Matrix_RotateY((this->actor.yawTowardsPlayer / 32768.0f) * 3.1415927f, MTXMODE_NEW);
```

Comment:
since that's the only spot in the file it has to be copypasted from this
https://github.com/zeldaret/oot/blob/master/src/overlays/effects/ovl_Effect_Ss_Fhg_Flash/z_eff_ss_fhg_flash.c#L107

the surrounding lines look suspiciously alike too lol